Makefile

```
@@ -0,0 +1,22 @@
+
+all: klaw api
+
+klaw:
+	mvn clean install
+	mvn clean package
```

This is not optimal. It effectively compiles and runs the tests twice. On top of that, `mvn clean package` is included in `mvn clean package`.
`mvn clean install` installs the package on your local maven repository, and unless we are dealing with a library, we probably don't need this.
I'd replace this 2 lines with the following:
```suggestion
	mvn clean verify
```

`mvn clean verify` Is what one usually wants: compilation, tests, `jar` file generation and integration test.